source/particle/interpolator/quadratic_least_squares.cc

```diff
@@ -210,25 +664,20 @@ namespace aspect
               {
-                prm.declare_entry ("Global particle property maximum",
-                                   boost::lexical_cast<std::string>(std::numeric_limits<double>::max()),
-                                   Patterns::List(Patterns::Double ()),
-                                   "The maximum global particle property values that will be used as a "
-                                   "limiter for the quadratic least squares interpolation. The number of the input "
-                                   "'Global particle property maximum' values separated by ',' has to be "
-                                   "the same as the number of particle properties.");
-                prm.declare_entry ("Global particle property minimum",
-                                   boost::lexical_cast<std::string>(std::numeric_limits<double>::lowest()),
-                                   Patterns::List(Patterns::Double ()),
-                                   "The minimum global particle property that will be used as a "
-                                   "limiter for the quadratic least squares interpolation. The number of the input "
-                                   "'Global particle property minimum' values separated by ',' has to be "
-                                   "the same as the number of particle properties.");
-                prm.declare_entry("Use limiter", "false",
-                                  Patterns::Bool (),
-                                  "Whether to apply a global particle property limiting scheme to the interpolated "
-                                  "particle properties.");
-
+                prm.declare_entry("Use quadratic least squares limiter", "true",
+                                  Patterns::List(Patterns::Bool()),
+                                  "Limit the interpolation of particle properties onto the cell, so "
+                                  "the value of each property is no smaller than its minimum and no "
+                                  "larger than its maximum on the particles of each cell, and the "
+                                  "average of neighboring cells. If more than one value is given, "
+                                  "it will be treated as a list. Cannot be used in combination with 'Use limiter'");
+                prm.declare_entry("Use boundary extrapolation", "false",
+                                  Patterns::List(Patterns::Bool()),
+                                  "Extends the range used by 'Use quadratic limiter' by linearly "
+                                  "interpolating values at cell boundaries from neighboring cells. "
+                                  "If more than one value is given, it will be treated as a list. "
+                                  "Enabling 'Use boundary extrapolation' without also enabling "
+                                  "'Use quadratic least squares limiter' does nothing.");
```

Comment:
Please change this behavior to throw an assert instead. We try to avoid having parameters that do not do anything unless another parameter is set.